5.rb

```diff
@@ -0,0 +1,19 @@
+def remove_first_last_two_char(str)
+    result = ""
+    len = str.length
+
+    if len <= 4
+        return result
```

Comment:
use single line condition
```suggestion
  return str if str.length <= 4
```